geckodriver.yaml

```
@@ -0,0 +1,42 @@
+package:
+  name: geckodriver
+  version: 0.35.0
+  epoch: 0
+  description: A tool for exploring each layer in a docker image
+  copyright:
+    - license: MPL-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - openssl-dev
```

let's build it using cargo-auditable as per our best practices. 
```suggestion
      - busybox
      - cargo-auditable
      - openssl-dev
```